.stylelintrc

```diff
@@ -5,7 +5,7 @@
   ],
   "ignoreFiles": ["static/sass/_hljs.scss"],
   "rules": {
-    "order/properties-alphabetical-order": true,
+    "order/properties-alphabetical-order": null,
```

Comment:
Could we not remove this rule entirely and subsequently the 'stylelint-order' plugin? It seems like it isn't being used